service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/utils/DateFormatUtil.java

```diff
@@ -13,12 +13,24 @@
         .appendPattern("yyyyMMddHHmmss")
         .toFormatter();
 
+    private static final DateTimeFormatter TEXT_DATE_TIME_FORMATTER = new DateTimeFormatterBuilder()
+        .appendPattern("yyyy-MM-dd HH:mm:ss")
+        .toFormatter();
+
     public static String formatDate(Date date) {
+        return format(date, DATE_TIME_FORMATTER);
+    }
+
+    public static Object formatTextDate(Date date) {
+        return format(date, TEXT_DATE_TIME_FORMATTER);
```

Comment:
The date in text should have different format date then output date. For example: 2010-01-13 14:45:40